packages/android_alarm_manager/example/pubspec.yaml

```diff
@@ -21,3 +21,7 @@ dev_dependencies:
 
 flutter:
   uses-material-design: true
+
+environment:
+  sdk: ">=2.1.0 <3.0.0"
+  flutter: ">=1.12.13+hotfix.5 <2.0.0"
```

Comment:
For the Flutter SDK constraint, please add the lower constraint only. See https://github.com/flutter/plugins/pull/3236 and https://github.com/flutter/flutter/pull/68437

Reply:
The goal of this PR is to get flutter/plugins CI green with tip-of-tree framework. Other cleanup work can happen in a subsequent PR.

Reply:
Why would we add a constraint in this PR, and then remove it again later? I don't understand the value in incurring additional debt. The `flutter` constraint does nothing wrt. getting CI green; you just need the Dart constraint for that.

Reply:
This PR is consistent with the rest of the repo. Fixing every pubspec in the repo is out of scope for this PR. Choosing the right time to do other cleanups is up to @amirh to decide.